.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,94 @@
+name: Deploy jobs.zooniverse.org Production
+
+on:
+  # Run this workflow on push to production-release tag (via chatops)
+  push:
+    tags:
+      - production-release
+
+  # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+
+
+jobs:
+  deploy_production:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - run: npm install
```

Comment:
Not a blocking issue but `npm ci` would be more secure for deploys here. 
https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
This repo doesn't even have a `package-lock.json` yet 😞 . Per the linked docs it notes "The project must have an existing package-lock.json or npm-shrinkwrap.json." There's discussion about refactoring this repo so at minimum it doesn't use `haw`, and certainly should use `package-lock.json`, but I think it's ok to pass on those larger refactors for now.

Reply:
D’oh! I missed that. Yeah, you can’t freeze the dependencies if there’s no lock file.

Reply:
I really really should (will!) add the lockfile asap, regardless of larger refactors. I'm scared to touch more than one thing at a time in this repo though, so doing things kinda crazily/overly small and incrementally.